app/Http/Requests/PetRequest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class PetRequest extends FormRequest
+{
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    public function rules(): array
+    {
+        return [
+            "name" => ["required", "string", "max:255"],
+            "species" => ["required", "string", "max:255"],
+            "breed" => ["nullable", "string", "max:255"],
+            "gender" => ["required", "string", "max:255"],
+            "age" => ["nullable", "string", "max:255"],
+            "size" => ["nullable", "string", "max:255"],
+            "weight" => ["nullable", "numeric"],
+            "color" => ["nullable", "string", "max:255"],
+            "sterilized" => ["nullable", "boolean"],
+            "description" => ["required", "string"],
+
+            "health_status" => ["nullable", "string"],
```

Comment:
What if user manipulates request data and provides extra long string? Provide max size everywhere you have string.